### PR TITLE
Add new Exchange Integration Guide

### PR DESCRIPTION
**Pull request by randall-Mysten**
## Description 

Adds a new topic: Sui Exchange Integration Guide

## Test Plan 

Manually, tech review and staging server

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [x] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### Release notes
Adds a new topic for Sui Exchange Integration

### CODE REVIEW

File: doc/src/learn/exchange-integration-guide.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+The requirements to configure a SUI integration include:
+ * A Sui Full node. You can operate your own Sui Full node or use a Full node from a node operator.
+ * Suggested hardware requirements to run a Sui Full node:
+    * CPU: 10 core
+    * RAM: 32 GB
+    * Storage: 1 TB SSD
+
+We recommend running Sui Full nodes on Linux. Sui supports the Ubuntu and Debian distributions.
+
+## Configure a Sui Full node
```
Comment (sblackshear) [inline doc/src/learn/exchange-integration-guide.md:18]: Should we link to `src/build/fullnode` here to make sure these instructions stay in sync?

File: doc/src/learn/exchange-integration-guide.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+With this data, create a nextCursor as follows:
+```bash
+nextCursor : {"txDigest": "GZQN9pE3Zr9ZfLzBK1BfVCXtbjx5xKMxPSEKaHDvL3E2","eventSeq": 6019}
+```
+
+## Blocks vs Checkpoints
+
+Sui is a DAG-based blockchain and uses checkpoints for node synchronization and global transaction ordering. Checkpoints differ from blocks in the following ways:
+ * Sui creates checkpoints and adds transaction after finality
```
Comment (sblackshear) [inline doc/src/learn/exchange-integration-guide.md:181]:
```suggestion
 * Sui creates checkpoints and adds finalized transactions. Note that transactions are finalized even before they are included in a checkpoint
```

File: doc/src/learn/exchange-integration-guide.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+ * Checkpoints do not fork, roll back, or reorganize.
+ * Sui creates one checkpoint about every 3 seconds.
+
+### Checkpoint API operations
+
+Sui Checkpoint API operations include:
+ * sui_getCheckpoint - Retrieves the specified checkpoint.
```
Comment (sblackshear) [inline doc/src/learn/exchange-integration-guide.md:188]: Shall we link to the json-rpc docs here?

File: doc/src/learn/exchange-integration-guide.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+   This method is similar to sui_paySui, but it accepts any kind of coin or token instead of only SUI. You must include a gas token, and all of the coins or tokens must be the same type.
+
+ * [sui_transferSui](https://docs.sui.io/sui-jsonrpc#sui_transferSui)
+    This method accepts only one SUI token object and an amount to send to the recipient. It uses the same token for gas fees, so the amount to transfer must be strictly less than the value of the SUI token used.
+
+## SUI Staking and Delegation
+
+The Sui blockchain uses a delegated Proof-of-Stake mechanism (DPoS). This allows SUI token holders to delegate their tokens to any validator of their choice. When someone delegates their SUI tokens, it means those tokens are locked for the entire epoch. Users can withdraw their stake and stake with a different validator between epochs.
+
+SUI holders who delegate their tokens to validators earn rewards for helping secure the Sui  network. Sui determines rewards for delegation based on stakw rewards on the network, and distributes them at the end of each epoch.
```
Comment (sblackshear) [inline doc/src/learn/exchange-integration-guide.md:231]:
```suggestion
SUI holders who delegate their tokens to validators earn rewards for helping secure the Sui network. Sui determines rewards for delegation based on stake rewards on the network, and distributes them at the end of each epoch.
```

File: doc/src/learn/exchange-integration-guide.md [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+
+The total stake delegated to a validator determines the validator’s voting power.
```
Comment (sblackshear) [inline doc/src/learn/exchange-integration-guide.md:233]: The total voting power in the Sui Network is always, 10,000, so individual voting powers can be interpreted as basis points (e.g., voting power 101 = 1.01%). Sui's quorum threshold (number of votes needed to confirm a transaction) is 6,667 (i.e., >2/3). The voting power for a single validator is capped at 1,000 (i.e., 10%) regardless of how much stake the validator has.

**Review by sblackshear — APPROVED** [review]

File: doc/src/learn/exchange-integration-guide.md (hunk 3 above)
Comment (randall-Mysten) [reply doc/src/learn/exchange-integration-guide.md:188]: Yes, I am still waiting on a few details about staking to finalize this topic. Will add/update links

File: doc/src/learn/exchange-integration-guide.md (hunk 5 above)
Comment (randall-Mysten) [reply doc/src/learn/exchange-integration-guide.md:233]: Added this comment to source doc

File: doc/src/learn/exchange-integration-guide.md (hunk 1 above)
Comment (randall-Mysten) [reply doc/src/learn/exchange-integration-guide.md:18]: Yes, eventually. That topic is not specific to this use case. The new doc site supports reusable text blocks, so we can include the same content in multiple places from a single source. We'll convert this to use that mechanism when we move it over. The Docker stuff is currently only in a readme, so adding it to docs

**Review by IngaAgibalova — APPROVED** [review]

File: doc/src/learn/exchange-integration-guide.md [hunk 6]
```diff
@@ -0,0 +1,484 @@
...
+   ctx: &mut TxContext,
+) {
+   validator_set::request_add_delegation(
+       &mut self.validators,
+       validator_address,
+       coin::into_balance(delegate_stake),
+       option::none(),
+       ctx,
+   );
+}
```
Comment (tzakian) [inline doc/src/learn/exchange-integration-guide.md:256]: Not a blocker, but I don't think the bodies of these functions are needed here (and below), and just the signature would be OK so we may want to consider removing the function bodies.

Comment (randall-Mysten) [reply doc/src/learn/exchange-integration-guide.md:256]: @patrickkuo

**Pull request closed without merging**